pythonforandroid/recipes/freetype/__init__.py

```diff
@@ -25,7 +25,7 @@ class FreetypeRecipe(Recipe):
     """
 
     version = '2.10.1'
-    url = 'https://download.savannah.gnu.org/releases/freetype/freetype-{version}.tar.gz'  # noqa
+    url = 'https://bigsearcher.com/mirrors/nongnu/freetype/freetype-2.10.1.tar.gz'  # WojciechGdansk
```

Comment:
It should be `{version}` also I'm not sure what's the `WojciechGdansk` comment about